src/class-parsely.php

```diff
@@ -608,7 +608,12 @@ private function get_managed_credentials() {
 		}
 
 		$credentials = apply_filters( 'wp_parsely_credentials', array() );
-		$result      = array();
+
+		if ( ! is_array( $credentials ) || 0 === count( $credentials ) ) {
```

Comment:
Can we use empty instead of `0 === count`?

Reply:
This was done in the past, but at some point all `empty()` occurrences were removed. From then on, checks were updated to be as specific as possible and this has become a convention for this project. If you search the code, you'll see that `empty()` is not being used anywhere. Generally, many people consider `empty()` problematic and there are articles about that on the web.